interop/authzen-search-demo/app/data/pdps.server.ts

```diff
@@ -0,0 +1,13 @@
+const pdps = [
+  {
+    name: "Cerbos",
+    host: "https://authzen-proxy-demo.cerbos.dev",
+  },
+  {
+    name: "Axiomatics",
+    host: "https://...",
+    auth: "Bearer somevalue",
```

Comment:
How do we want to handle this?  backend secrets that are injected in as environment variables?

Reply:
Yup once we have some PDPs that we can test against, then this will read from `PROCESS.ENV....`